frameworks/cassandra/tests/test_recovery.py

```diff
@@ -11,6 +11,8 @@
 import sdk_tasks
 import sdk_utils
 
+TIMEOUT_SECONDS = 20 * 60
```

Comment:
Is this timeout only used while waiting for recovery? If so, either rename it, or specify it closer to the point where it is used.